### PR TITLE
Add end-user documentation for new MariaDB service

### PR DESCRIPTION
**Pull request by TheBigLee**
This PR will add the end-user documentation for the new MariaDB service.

### CODE REVIEW

File: docs/modules/ROOT/pages/vshn-managed/mariadb/maintenance.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+The day and time can be adjusted as described below. Times are in the CET time zone.
+
+.Redis instance with custom maintenance
+[source,yaml]
+----
+apiVersion: vshn.appcat.vshn.io/v1
+kind: Redis
+metadata:
+  name: redis-app1-prod
```
Comment (wejdross) [inline docs/modules/ROOT/pages/vshn-managed/mariadb/maintenance.adoc:15]: file is named mariadb/maintenance but contains Redis values

File: docs/modules/ROOT/pages/vshn-managed/redis/maintenance.adoc [hunk 2]
```diff
@@ -2,17 +2,17 @@
 
 By default, there will be a security maintenance on Tuesday night (Central Europe Time).
 
-NOTE: For more information about maintenance procedures, please visit https://products.docs.vshn.ch/products/appcat/redis.html#_recurring_maintenance[the product description].
+NOTE: For more information about maintenance procedures, please visit https://products.docs.vshn.ch/products/appcat/mariadb.html#_recurring_maintenance[the product description].
```
Comment (wejdross) [inline docs/modules/ROOT/pages/vshn-managed/redis/maintenance.adoc:5]: file is called redis/maintenance but contains mariadb references

**Review by wejdross — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/vshn-managed/mariadb/maintenance.adoc (hunk 1 above)
Comment (TheBigLee) [reply docs/modules/ROOT/pages/vshn-managed/mariadb/maintenance.adoc]: Fixed

File: docs/modules/ROOT/pages/vshn-managed/redis/maintenance.adoc (hunk 2 above)
Comment (TheBigLee) [reply docs/modules/ROOT/pages/vshn-managed/redis/maintenance.adoc]: Fixed

File: docs/modules/ROOT/pages/vshn-managed/mariadb/backup.adoc [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+spec:
+  parameters:
+    ...
+    backup:
+      schedule: 0 22 * * * <1>
+    ...
+----
+<1> The schedules for your backups.
+
+== Listing Backups
```
[the remark this answers is not in the document]
Comment (TheBigLee) [reply docs/modules/ROOT/pages/vshn-managed/mariadb/backup.adoc]: I've removed it again

File: docs/modules/ROOT/pages/vshn-managed/mariadb/create.adoc [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+apiVersion: vshn.appcat.vshn.io/v1
+kind: VSHNMariaDB
+metadata:
+  name: mariadb-app1-prod # <1>
+  namespace: prod-app # <2>
+spec:
+  parameters:
+    service:
+      version: "11.2" # <3>
+      mariadbSettings: | # <4>
```
[the remark this answers is not in the document]
Comment (TheBigLee) [reply docs/modules/ROOT/pages/vshn-managed/mariadb/create.adoc]: Removed it

File: docs/modules/ROOT/pages/vshn-managed/mariadb/delete.adoc [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+$ oc delete vshnmariadb.vshn.appcat.vshn.io mariadb-app1-prod
+----
+
+[IMPORTANT]
+.Non-reversible operation
+====
+If you delete the `VSHNMariaDB` object from the cluster, it will be deleted together with all of its data!
+====
+
+== Delete protected MariaDB instance
```
[the remark this answers is not in the document]
Comment (TheBigLee) [reply docs/modules/ROOT/pages/vshn-managed/mariadb/delete.adoc:18]: dito

File: docs/modules/ROOT/pages/vshn-managed/mariadb/plans.adoc [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+[WARNING]
+.APPUiO Cloud Fair Use Policy
+====
+On APPUiO Cloud, the https://docs.appuio.cloud/user/explanation/fair-use-policy.html[Fair Use Policy] also applies to all AppCat Services by VSHN.
+You will be billed extra for CPU requests that exceed the memory to CPU ratio.
+
+The default plans all adhere to the Fair Use Policy.
+====
+
+== Node Selectors
```
Comment (Kidswiss) [inline docs/modules/ROOT/pages/vshn-managed/mariadb/plans.adoc:111]: is that actually implemented?

**Review by Kidswiss — CHANGES_REQUESTED** [review]
Just one more thing, then LGTM

**Review by Kidswiss — APPROVED** [review]

**Review by wejdross — APPROVED** [review]

**Pull request merged**